zone.go

```diff
@@ -44,6 +44,7 @@ type Zone struct {
 	Betas       []string `json:"betas"`
 	DeactReason string   `json:"deactivation_reason"`
 	Meta        ZoneMeta `json:"meta"`
+	Account     Account  `json:"account"`
```

Comment:
[In the CloudFlare API, a zone](https://api.cloudflare.com/#zone-properties)'s account is comprised of two strings: an ID and a name.  If we are going to redefine the Zone struct to include an Account element, it would seem wise to define it as a two-element struct, either defined inline (as with Host, which has Name and Website), or as a new Account type (as with Owner, which has ID, Email, and OwnerType).

Reply:
I think we can leave it as is. It's technically the same object (`/account == /zones's "account" field`), just returning a different subset of data.

Reply:
@patryk: Apologies.  I misread.  The Account type is already defined elsewhere with name and ID, and this change adds an Account of that type to the Zone type, which was the second of my proposed alternatives.